config/measureCoaddSources.py

```diff
@@ -1,8 +1,18 @@
+import os.path
+
+config.measurement.load(os.path.join(os.path.dirname(__file__), "apertures.py"))
+config.measurement.load(os.path.join(os.path.dirname(__file__), "kron.py"))
+config.measurement.load(os.path.join(os.path.dirname(__file__), "convolvedFluxes.py"))
+config.measurement.load(os.path.join(os.path.dirname(__file__), "hsm.py"))
+config.load(os.path.join(os.path.dirname(__file__), "cmodel.py"))
+
+# Set reference catalog for Gen2.
 config.match.refObjLoader.ref_dataset_name = "ps1_pv3_3pi_20170110"
-config.match.refObjLoader.filterMap['u'] = 'g'
-config.match.refObjLoader.filterMap['Y'] = 'y'
-config.match.refObjLoader.filterMap['N419'] = 'g'
-config.match.refObjLoader.filterMap['N540'] = 'g'
-config.match.refObjLoader.filterMap['N708'] = 'i'
-config.match.refObjLoader.filterMap['N964'] = 'z'
```

Comment:
Are you sure these aren't needed anymore? It is probably moot due to Gen2 going away, but the mapping is for non-ugrizy filters that need some "close enough" lookup refcat lookup filter.

Reply:
I moved the filter definitions to the file `config/filterMap.py`, as in `obs_subaru`, which seemed to be the most sensible thing to do. Broad band filters were removed because they were causing schema errors (see [this comment](https://github.com/lsst/obs_decam/pull/208#discussion_r754504808) from John P above, and [this Slack thread](https://lsstc.slack.com/archives/C2JPMCF5X/p1637601585397800) for my original report of this error).